ETC.tsv

```diff
@@ -1,9 +1,9 @@
 ETC_20150317_000001	Kepa
-ETC_20150317_000002	The Kepa is not a very strong monster. However, attacking it recklessly may bring you to tears.
+ETC_20150317_000002	The Kepa is not a strong monster. However, attacking it recklessly may bring you to tears.
 ETC_20150317_000003	Large Kepa
-ETC_20150317_000004	Humans sometimes hunt Large Kepa for food.
+ETC_20150317_000004	Sometimes there are people who are trying to eat Large Kepa. Don't let them.
 ETC_20150317_000005	Red Kepa
-ETC_20150317_000006	Most people are unaware that the Red Kepa is actually quite similar to the Kepa.
+ETC_20150317_000006	Despite the normal cognition, it is Red Kepa which is more similar to the original Kepa
```

Comment:
@ttgmichael / In the original text, they are comparing "the Red Kepa" to sth they didn't mention about. Let me explain the meaning of the original text; Compared to the creature they didn't mention about, the Red Kepas are "more simillar" to the original Kepas. Maybe they are talking about their look, maybe they are talking about genes, maybe they are talking about the nature of these creatures, but I don't know. How do you think we should?